ui/src/app/data/abstract-data.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from "@angular/core"
+import {HttpClient, HttpHeaders, HttpParams} from "@angular/common/http"
+import {AbstractService} from "../abstract.service"
+import {Router} from "@angular/router"
+import {Location} from "@angular/common"
+import {AuthService} from "../auth/auth-service"
+
+
+interface ApiGetParams {
```

Comment:
Is this "params for an HTTP GET" or " get the params for the request"?

Reply:
Oops, this interface is already in the abstract-service class that extended to this class, I forgot to remove it. Manuel already  pushed the change... resolving this thread.